src/CalcManager/Ratpack/conv.cpp

```diff
@@ -1058,10 +1016,10 @@ bool stripzeroesnum(_Inout_ PNUMBER pnum, int32_t starting)
     if (fstrip)
     {
         // Remove them.
-        memmove(pnum->mant, pmant, (int)(cdigits * sizeof(MANTTYPE)));
+        copy(pmant, pmant + cdigits, pnum.mant.begin());
```

Comment:
In case the elements overlap, the result of `std::copy` is undefined, unlike with `memmove`